### ISSUE TITLE
Hide battery percent when charging

### ISSUE THREAD

**Issue by jmarhold** [freakified/TimeStylePebble#11]
The current value of battery when charging is weird. E. g. watchface shows 90%, then i turn on charging and watchface shows 70%.Maybe add condition to  line 145 of main.c : if(Settings_showBatteryPct && !chargeState.is_charging) {}

**Comment by tilden** [comment]
I'm not sure what you're saying. That the percentage should always show when charging, regardless of the setting?

**Comment by jmarhold** [comment]
I am just saying that percentage has different value before and while charging. See photos - 90% vs 60%.
![normal](https://cloud.githubusercontent.com/assets/13431049/8870668/027f5944-31f0-11e5-8d7d-6db95870890a.png)
![charging](https://cloud.githubusercontent.com/assets/13431049/8870671/08522522-31f0-11e5-933a-343ed6452bf7.png)

Its little bit confusing,so i suppose to not show percentage while charging (pebble in settings also doesn't show percentage, only charging icon)

**Comment by tilden** [comment]
Oh, thanks for the photos, now I get it. I just verified it on my watch as well -- the Pebble Time really does show strange numbers when the watch is charging. However, I also tested on an original Pebble, and the charge numbers make sense.

Because of that, it looks like this is really a bug in the Pebble Time's firmware. I [posted on the developer forum](http://forums.getpebble.com/discussion/27918/battery-charge-percent-returning-crazy-numbers-when-charging-but-only-on-the-pebble-time) -- maybe someone from Pebble will see it and comment.

Edit: someone found an earlier Reddit post with an official response:
https://www.reddit.com/r/pebble/comments/3bodnk/full_charge_at_70/

Based on that, I'll remove the % when charging.
